clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3738,6 +3738,103 @@ behavior of the program is undefined.
   }];
 }
 
+def FormatMatchesDocs : Documentation {
```

Comment:
I'm hopeful someone comes along and does a better 'english/documentation-quality' review of this.  I don't have the mental acuity at the moment.